lambda-term.opam

```
@@ -10,7 +10,7 @@ build: [
   ["dune" "runtest" "-p" name "-j" jobs] {with-test}
 ]
 depends: [
-  "ocaml" {>= "4.02.3"}
+  "ocaml" {>= "4.03.0"}
```

```suggestion
  "ocaml" {>= "4.08.0"}
```
I just realized `Format.stag` and so on were added in OCaml 4.08 so this needs to change as well

---

You should now have access as soon as you accept it (check your email.)